README.md

```diff
@@ -62,3 +62,9 @@ email to ensure we received your original message. Further information, includin
 Copyright (c) Microsoft Corporation. All rights reserved.
 
 Licensed under the [MIT License](./LICENSE).
+
+## Forker
+Daniel L. Astillero
+daniel.l.astillero@gmail.com
++63917-390-2559
```

Comment:
you sure you want to commit this to the offical repository from microsoft?

Reply:
what even is this?

Reply:
a poor attempt at advertising himself

Reply:
yikes